### DISCUSSION TITLE
How to Remove Siege of Avalon from this repo!

### DISCUSSION THREAD

**Discussion by CartBlanche** [opengaming/osgameclones#1679] [General]
I recently received and message that Siege of Avalon was added to this repo. I do not know who you are, I was not consulted about this and do not want my game pulled into this repo. How can I get it removed from this repo?

**Comment by cxong** [comment]
Hi, Siege of Avalon was originally added in 2015, the github notification is from our bot since the game was recently updated (renamed).

OSGC is a community-curated list of open source clones and remakes of (older) games. It is a useful resource for both players looking to replay modern versions of older classics, or developers looking to tinker and study those games. You can go to osgameclones.com to read a longer description of the site.

If you don't mind me asking, why do you want the game removed from the repo? Do you feel that it is being misrepresented, and if so how can we improve it?
